animalfacts.py

```diff
@@ -369,6 +369,7 @@ def check_comment_for_animal(comment, reddit):
     botengine('whale', '\swhales?\s', reddit, WHALE_FACTS, comment)
     botengine('wolf', '\swol(f|ves)\s', reddit, WOLF_FACTS, comment)
     botengine('wolverine', '\swolverines?\s', reddit, WOLVERINE_FACTS, comment)
+    botengine('wombat', '\swombatss?\s', reddit, WOMBAT_FACTS, comment)
```

Comment:
@joel-g Will this work? Does it need the final "s" removing in wombats?

Reply:
I guess so..otherwise can you suggest me the change if possible ?..Thanks

Reply:
I am reasonably sure it should be '\swombats?s\' so that it matches the other animals. Take out the extra S?

Reply:
I'll fix this myself.